services/eth_gateway.go

```diff
@@ -137,12 +185,16 @@ var (
 	EthWrapper           Eth
 )
 
+// Actual transactions on etherscan, tends to be 53898
```

Comment:
Could remove this comment if you want, or change it to something like "Limits selected based on actual transactions from etherscan" or something.  Not a big deal though.